File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/OrderWeight.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class OrderWeight {

    @SerializedName("weight_raw")
    @Expose
    private Long weightRaw;
    @SerializedName("weight")
    @Expose
    private String weight;

    public Long getWeightRaw() {
        return weightRaw;
    }

    public void setWeightRaw(Long weightRaw) {
        this.weightRaw = weightRaw;
    }

    public String getWeight() {
        return weight;
    }

    public void setWeight(String weight) {
        this.weight = weight;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/OrderWeight.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class OrderWeight {

    @SerializedName("weight_raw")
    @Expose
    var weightRaw: Long? = null
    @SerializedName("weight")
    @Expose
    var weight: String? = null

}
